src/profile/d3Elevation.ts

```diff
@@ -504,19 +487,19 @@ function d3Elevation(options, functions) {
       g.select('.overlay').on('mouseout', mouseout).on('mousemove', mousemove);
 
       /**
-       * @this {d3.ContainerElement}
+       * @param event Event
        */
-      function mousemove() {
-        const mouseX = d3mouse(this)[0];
+      function mousemove(event: Event): void {
+        const mouseX = d3pointer(event)[0];
         const x0 = x.invert(mouseX);
 
         profile.highlight(x0);
       }
 
       /**
-       *
+       * Mouseout
```

Comment:
Well at this point, you can remove the comment.
Quick check also other comment. Remove them or better, improve them !

Reply:
I got a jsdoc error if I remove it, and a CI that fails ... Quite the same for all the useless ones in this file...